.github/workflows/edFi.common pullrequest.yml

```diff
@@ -0,0 +1,30 @@
+name: EdFi.Common Pull request build and test
+
+on:
+  pull_request:
+    branches: [main]
+
+env:
+  INFORMATIONAL_VERSION: "5.4"
+  BUILD_INCREMENTER: "10"
```

Comment:
I just realized, but since this is the pull request and no packages get made for this one, the env variable for it here, along with the passing of the value to the build script can be removed for this pull request and the edfi.ods.api pull request workflow.

Reply:
yes updated . Thanks